maraboupy/test/test_network.py

```diff
@@ -79,6 +79,19 @@ def test_disjunction_constraint():
     for var in network.inputVars[0]:
         assert(abs(vals1[var] - 1) < 0.0000001 or abs(vals1[var]) < 0.0000001)
 
+def test_sigmoid_constraint():
```

Comment:
Maybe move the test to test/test_onnx.py ?
And make the test to be like: https://github.com/NeuralNetworkVerification/Marabou/blob/30c1ba9f5fbed1ca8ff2cebbb1f1cc7941c7fde8/maraboupy/test/test_onnx.py#L26